include/tvm/build_module.h

```diff
@@ -170,6 +170,9 @@ TVM_DLL Target intel_graphics(const std::vector<std::string>& options =
 TVM_DLL Target stackvm(const std::vector<std::string>& options =
                       std::vector<std::string>());
 
+/*! \return A target for ext_dev */
+TVM_DLL Target ext_dev(const std::vector<std::string>& options =
```

Comment:
Shall we avoid using the name `ext_dev` for a target, since there is a `ext_dev` for TVMContext? See

https://github.com/apache/incubator-tvm/blob/97e333cac8313075e7f7067d2d078782e793e0cc/python/tvm/ndarray.py#L171

Reply:
Yes, same thing used by opencl and metal. But I agree, `ext` might be a better name.